src/pages/TeamPage/index.js

```diff
@@ -0,0 +1,349 @@
+import React, { useState, useEffect } from "react";
+import {
+  Button,
+  Typography,
+  Box,
+  Container,
+  Stack,
+  CssBaseline,
+  Card,
+} from "@mui/material";
+import { Grid } from "@mui/material";
+import useMediaQuery from "@mui/material/useMediaQuery";
+import useStyles from "./style";
+import axios from "axios";
+import Tippy from "@tippyjs/react";
+import "./style.css";
+import { breakpoints } from "../../theme/constant";
+import LinkedIn from "../../components/common/SocialMediaIcons/LinkedIn";
+import Twitter from "../../components/common/SocialMediaIcons/Twitter";
+
+/**
+ *
+ * @param {Array} arr
+ * mutates Array inplace
+ */
+function shuffleArray(arr) {
+  for (let i = arr.length - 1; i >= 0; i--) {
+    const newIndex = Math.floor(Math.random() * (i + 1));
+    [arr[i], arr[newIndex]] = [arr[newIndex], arr[i]];
+  }
+  return arr;
+}
+
+function TeamPage() {
+  useEffect(() => {
+    axios({
+      url: `https://anandpatel504.github.io/tarabai-shinde/data/ng_team.json`,
+    }).then((res) => {
+      setTeam(res.data);
+    });
+  }, []);
+  const [team, setTeam] = useState([]);
+  const [members, setMembers] = useState({
+    teamMembers: true,
+    volunteers: false,
+  });
```

Comment:
Since `teamMembers` and `volunteers` are always opposite Boolean values, do you really need both of them?
```suggestion
  const [showTeam, setShowTeam] = useState(true);
```